ddclient

```diff
@@ -1988,128 +1993,133 @@ EOM
 ## geturl
 ######################################################################
 sub geturl {
-    my $proxy    = shift || '';
-    my $url      = shift || '';
-    my $login    = shift || '';
-    my $password = shift || '';
-    my $headers  = shift || '';
-    my $method   = shift || 'GET';
-    my $data     = shift || '';
+    my ($p) = @_;
```

Comment:
I am not a big fan of the single p variable. Maybe we find a good three letter one?

Reply:
You haven't written much Go code, have you? :stuck_out_tongue_winking_eye: 

I decided to keep the original variables.